examples/suit_update/tests/01-run.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2019 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import subprocess
+import sys
+import tempfile
+
+from testrunner import run
+
+# Default test over loopback interface
+COAP_HOST = "[fd00:dead:beef::1]"
+
+UPDATE_TIMEOUT = 120
+MANIFEST_TIMEOUT = 15
+
+USE_ETHOS = int(os.getenv("USE_ETHOS", "1"))
+TAP = os.getenv("TAP", "tap0")
+
+
+def start_aiocoap_fileserver():
+    tmpdirname = tempfile.TemporaryDirectory()
+    aiocoap_process = subprocess.Popen(
+        "exec aiocoap-fileserver %s" % tmpdirname.name, shell=True
+    )
+
+    return tmpdirname, aiocoap_process
+
+
+def cleanup(tmpdir, aiocoap_process):
+    aiocoap_process.kill()
```

Comment:
This doesn't kill everything properly, specificity fails to clean up the created `tap` interface. SIGKILL is killing all child processes but not removing the interface, this is because in the script:

`trap "cleanup" INT QUIT TERM EXIT`.

Can you tell if there is a reason for this? Or should we push the change in master? I'll take a look at the history.

Reply:
Ah, the shell script cannot trap "KILL". Need to change to send "TERM" first, then "KILL" a little later...

Reply:
sorry, this was outdated, with setup_network.sh, this isn't an issue anymore